src/fidasim.f90

```diff
@@ -254,6 +254,14 @@ module libfida
         !+ Torodial magnetic field [T]
     real(Float64) :: bz = 0.d0
         !+ Vertical magnetic field [T]
+
+    real(Float64) :: dbr_dr = 0.d0
+    real(Float64) :: dbr_dz = 0.d0
+    real(Float64) :: dbt_dr = 0.d0
+    real(Float64) :: dbt_dz = 0.d0	
+    real(Float64) :: dbz_dr = 0.d0
+    real(Float64) :: dbz_dz = 0.d0	
+	
```

Comment:
This section needs the inline documentation so it will show up on the website. 

Documentation comments are preceded with a `!+`. Like this
```fortran
real(Float64) :: dbr_dr = 0.d0
    !+ Partial derivative of radial magnetic field with respect to r
```